### ISSUE TITLE
tune: module 'tensorflow' has no attribute __version__ in Ray Trainable since v0.7.7

### ISSUE THREAD

**Issue by Nithanaroy** [ray-project/ray#8729]
### What is the problem?

*Ray version and other system information (Python version, TensorFlow version, OS):*

- Verified on Ray >=0.7.7
- Tensorflow 1.15
- CentOS 7

I get `module 'tensorflow' has no attribute __version__` error when I run my code using `run` API from Ray Tune. It works fine when I manually call `myTrainable.train()`. This only appears since v0.7.7

### Reproduction (REQUIRED)
Please provide a script that can be run to reproduce the issue. The script should have **no external library dependencies** (i.e., use fake or mock data / environments):

I understand this is a requirement, but I have complicated proprietary script which would be hard to obfuscate without losing important details. I'll try to describe the problem here and I'm assuming you are already familiar with this problem as it looks quite common to appear. Here is the structure of my trainable,

```python
from mydataprocessor import process_data

class MyTrainable(Trainable):
  def _setup(self, config):
    process_data(...)
    ...
 ...
```
Here is the structure of my `mydataprocessor.py` module
```python
import tensorflow as tf

print(f"Working with Tensorflow v{tf.__version__}")

def process_data(...):
  ...
```

If we cannot run your script, we cannot fix your issue.
- [x] I have verified my script runs in a clean environment and reproduces the issue.
- [x] I have verified the issue also occurs with the [latest wheels](https://docs.ray.io/en/latest/installation.html).

**Comment by richardliaw** [comment]
Can you again try moving the tensorflow import into the training function scope?

**Comment by Nithanaroy** [comment]
Yeah, it works that way but is that the only solution? In v0.7.6 this
additional import wasn't necessary.

--
Nitin @ LinkedIn <http://bit.ly/nitinpasumarthy>

**Comment by cszhu** [comment]
Hello! This P3 issue has seen no activity in at least a year. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
